nomos-da/kzgrs-backend/src/common/mod.rs

```diff
@@ -96,21 +104,24 @@ impl AsRef<[Chunk]> for Column {
 }
 
 impl ChunksMatrix {
+    #[must_use]
     pub fn len(&self) -> usize {
         self.0.len()
     }
+    #[must_use]
```

Comment:
I see the point, but this feels that is more a spam than other thing :sweat_smile:

Reply:
I'm not against it. But it kinda tickles my brain when we have to do something repetitive. This should probably be some kinda flag instead of a directive to use manually. But lets see what the guys opinion is.